client/src/Movies/UpdateMovie.js

```diff
@@ -0,0 +1,100 @@
+import React, {useState, useEffect} from 'react';
+import api from '../utils/api'
+
+const UpdateMovie = (props) => {
+    const [movie, setMovie] = useState({
+        id: '',
+        title: '',
+        director: '',
+        metascore: '',
+        stars: []
+    })
+
+    useEffect(() => {
+        api().get(`/movies/${props.match.params.id}`)
+            .then(result => {
+                setMovie(result.data)
+            })
+            .catch(error => {
+                console.log(error)
+            })
+    }, [props.match.params.id])
+
+    const handleChange = e => {
+        setMovie({
+            ...movie,
+            [e.target.name]: e.target.value
+        })
+    }
+
+    const handleStars = e => {
+        setMovie({
+            ...movie,
+            stars:[e.target.value]
```

Comment:
I see that you implemented a way to handle the stars, but what is the effect?  What happens when you edit a star? Are they still in multiple lines or does it go into one line?